### ISSUE TITLE
Non-standard language tags for "Belarusian" and "English (Pakistan)" locales

### ISSUE THREAD

**Issue by alx-mag** [datafaker-net/datafaker#595]
Faker uses tags `by` and `en-PAK` for these locales. But the standard ones are `be` (or `be-BY`) and `en-PK`, see the [standard locales list](https://www.localeplanet.com/icu/index.html). These non-standard tags are usable with Faker but Java does not recognize them:
```
import java.util.Locale;

public class Test {

    public static void main(String[] args) {
        System.out.println(getLocaleDisplayName("by"));     // by
        System.out.println(getLocaleDisplayName("be-BY"));  // Belarusian (Belarus)
        System.out.println(getLocaleDisplayName("be"));     // Belarusian

        System.out.println(getLocaleDisplayName("en-PAK")); // pak
        System.out.println(getLocaleDisplayName("en-PK"));  // English (Pakistan)
    }

    static String getLocaleDisplayName(String languageTag) {
        return Locale.forLanguageTag(languageTag)
                .getDisplayName(Locale.ENGLISH);
    }
}
```
I think there should be added `be.yml` (or `be-BY.yml`) and `en-PK.yml`. The `by.yml` and `en-PAK.yml` can be left for backward compatibility.

**Comment by bodiam** [comment]
Thanks for reporting! Would you be willing to do a PR for this? We could deprecate the non standard ymls, and remove them next release

**Comment by bodiam** [comment]
I guess this can be closed now that #598 has been merged.
